web/views/datasets.py

```diff
@@ -6,7 +6,7 @@
 
 from core.forms import DatasetForm
 from core.forms.dataset import DatasetFormEdit
-from core.models import Dataset
+from core.models import Dataset, Exposure
 from core.models.utils import COMPANY
 from core.permissions import permission_required, CheckerMixin
```

Comment:
`permission_required` is not used in the file.

Neither are `redirect`, `get_object_or_404`, `user_passes_test`, `is_data_steward`

Reply:
removed unused imports